src/Chapter2.hs

```diff
@@ -852,14 +858,13 @@ To defeat the boss, implement the reversing function that takes a list
 and reverses it.
 
 >>> rewind [1 .. 5]
-[5,4,3,2,1]
-
 ♫ NOTE: The Haskell standard library already provides the "reverse"
   function, but in this task, you need to implement it manually. No
   cheating!
 -}
-rewind = error "rewind: Not Implemented!"
-
+rewind :: [Int] -> [Int]
+rewind (x:xs) = rewind xs ++ [x]
```

Comment:
Your solution is correct! However, it is slow. In lists, it is quite slow to add anything at the end of the list. That is why it is always better to rewrite it with the `:` cons. Remember the explanation with trains? 🚂 🚋 🚋 

That is why a more efficient solution is with the accumulator and the recursive function that will do the addition at the start of the list which is instant!

You can read a bit more about the `go` pattern in here: https://kowainik.github.io/posts/haskell-mini-patterns#recursive-go